nzsl-dictionary-scripts/env-production/main.tf

```
@@ -0,0 +1,71 @@
+
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "4.13.0"
```

suggestion: would be best not to pin to an exact version:

```suggestion
      version = "~> 4.13"
```